src/status_im/ui/components/desktop/events.cljs

```diff
@@ -1,19 +1,37 @@
 (ns status-im.ui.components.desktop.events
   (:require [status-im.utils.handlers :as handlers]
+            [status-im.utils.fx :as fx]
             [status-im.utils.platform :as platform]))
 
+(fx/defn change-tab
+  [{:keys [db]} tab-name]
+  {:db (assoc-in db [:desktop/desktop :tab-view-id] tab-name)})
+
+(fx/defn navigate-to
+  [{:keys [db]} tab-name]
+  {:dispatch [:navigate-to (if (and (= tab-name :home) (:current-chat-id db))
```

Comment:
Why not use navigate-to-cofx FN directly?

Reply:
@yenda yes, changing it to `navigate-to-cofx`